src/resolvers/ens.ts

```diff
@@ -10,9 +10,9 @@ export default async function resolve(name: string) {
       return false;
     }
 
-    const url = await ensResolver.getText('avatar');
+    let url = await ensResolver.getText('avatar');
     if (!url || !url.startsWith('http')) {
-      return false;
+      url = `https://metadata.ens.domains/mainnet/avatar/${name}`;
```

Comment:
Not sure if there is some rate limit 😄 but we can try

Reply:
We're using this as fallback, only on NFT image, should be fine